src/test/kotlin/com/atlassian/performance/tools/hardware/IntegrationTestRuntime.kt

```diff
@@ -17,7 +18,7 @@ import java.util.*
 
 object IntegrationTestRuntime {
 
-    const val taskName = "QUICK-132-fix-v3"
+    const val taskName = "DCNG-195-initial"
```

Comment:
Same comment as above. Why do we keep over-writing this rather than having a test per taskName ?

Reply:
You're right, we should be making new ones instead of mutating old ones